pkg/querier/http.go

```diff
@@ -409,6 +409,11 @@ func (q *QuerierAPI) validateMaxEntriesLimits(ctx context.Context, expr syntax.E
 	return nil
 }
 
+// DetectedLabelsHandler returns a response for detected labels
+func (q *QuerierAPI) DetectedLabelsHandler(ctx context.Context, concrete *logproto.DetectedLabelsRequest) (*logproto.DetectedLabelsResponse, error) {
+	return nil, nil
```

Comment:
can you return some classic labels for now so at least while you iterate it works for most workload ?

`cluster`,`namespace`,`instance` ?

Reply:
yes, sure can do that